zeppelin-web/src/app/jobmanager/jobmanager.html

```diff
@@ -34,7 +34,16 @@ <h3 class="new_h3">
   <div class="row">
     <div class="col-md-6 text-left">
       <div class="form-inline">
-        <span class="labelBtn btn-group" style="margin-left: 0px; padding-left: 10px;">
+        <span class="labelBtn btn-group">
+          <button type="button"
+                  class="btn btn-default"
+                  style="width: 25px; height: 25px; margin-right: 0px; padding: 1px 0px 3px 3px"
+                  ng-click="onChangeSortAsc()"
+                  tooltip-placement="right" tooltip="Sort by Note name">
```

Comment:
@cloverhearts With current status, it shows same tooltip msg for different two cases. 
<img src="https://cloud.githubusercontent.com/assets/10060731/21747846/6c348586-d5b8-11e6-9624-ef24a2b64a10.gif" width="300px">

So how about making this msg more self-descriptive? for example 

 - for `fa-sort-amount-asc` icon -> "Sort notes by desc order" 
 - for `fa-sort-amount-desc` icon -> "Sort notes by asc order"